mitodl/odl-video-service
# Dorska/analytics overlay

#### What are the relevant tickets?
Closes #551 .

#### What's this PR do?
Displays analytics as an overlay on top of the video.

#### How should this be manually tested?
1. Find a video for which you have (a) analytics data, and (b) edit access .
2. Go to the detail page for the video.
3. Click the `show/hide analytics` button. An overlay should pop up over the video.
4. The overlay should show a loading message at first.
5. The overlay should then show an analytics chart with a time slider.
6. Click on the time slider: the video time should change, and the analytics info table should show data for the updated time.
7. Start playing the video. The analytics overlay should update as the video time changes.
8. Click the `show/hide analytics` button again. The overlay should disappear.

Confirm that the chart and table look reasonable for:
1. A video with singlecam analytics
2. A video with multicam analytics.
